code/datums/brain_damage/psychological.dm

```diff
@@ -0,0 +1,297 @@
+/*TODO:
+	PTSD
+*/
+/datum/brain_trauma/psychological
```

Comment:
Isn't there a variable that you can set to keep these psychological brain traumas from showing up from brain damage (like other kinds of traumas would)?

Reply:
Aha! Found it; set random_gain to FALSE to keep people from developing schizophrenia (instead of some other trauma) from a toolbox blow to the head while their psych instability is at -99.